CMakeModules/build_forge.cmake

```diff
@@ -36,7 +36,7 @@ ExternalProject_Add(
       -DCMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH_ALT_SEP}"
       -DBUILD_SHARED_LIBS:BOOL=ON
       -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>
-      -DCMAKE_BUILD_TYPE:STRING=Release
+      -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}
```

Comment:
I think its its appropriate to keep this as release. We are using the library not debugging it.